src/parser/core/modules/Weaving.tsx

```diff
@@ -217,38 +247,57 @@ export class Weaving extends Analyser {
 			return false
 		}
 
-		const panels = this.badWeaves.map(item => ({
-			key: item.leadingGcdEvent?.timestamp ?? this.parser.pull.timestamp,
-			title: {
-				content: <>
-					<strong>{this.parser.formatEpochTimestamp(item.leadingGcdEvent?.timestamp ?? this.parser.pull.timestamp)}</strong>
-					&nbsp;-&nbsp;
-					<Plural
-						id="core.weaving.panel-count"
-						value={item.weaves.length}
-						_1="# weave"
-						other="# weaves"
-					/>
-					&nbsp;(
-					{this.parser.formatDuration(item.gcdTimeDiff)}
-					&nbsp;
-					<Trans id="core.weaving.between-gcds">between GCDs</Trans>
-					)
-				</>,
-			},
-			content: {
-				content: <Rotation events={[
-					...(item.leadingGcdEvent != null ? [item.leadingGcdEvent] : []),
-					...item.weaves,
-				]}/>,
-			},
-		}))
-
-		return <Accordion
-			exclusive={false}
-			panels={panels}
-			styled
-			fluid
-		/>
+		return <Table unstackable collapsing>
+			<Table.Header>
+				<Table.Row>
+					<Table.HeaderCell collapsing>
+						<strong><Trans id="core.weaving.table.time">Time</Trans></strong>
+					</Table.HeaderCell>
+					<Table.HeaderCell collapsing>
+						<strong><Trans id="core.weaving.table.weave-info">Weave info</Trans></strong>
+					</Table.HeaderCell>
+					<Table.HeaderCell>
+						<strong><Trans id="core.weaving.table.weave-actions">Actions</Trans></strong>
+					</Table.HeaderCell>
+				</Table.Row>
+			</Table.Header>
+			<Table.Body>
+				{
+					this.badWeaves.map((item) => {
+						return <Table.Row key={item.leadingGcdEvent.timestamp}>
+							<Table.Cell textAlign="center">
+								<span style={{marginRight: 5}}>{this.parser.formatEpochTimestamp(item.leadingGcdEvent.timestamp)}</span>
+								<Button
+									circular
+									compact
+									size="mini"
+									icon="time"
+									onClick={() => this.timeline.show(item.leadingGcdEvent.timestamp - this.parser.pull.timestamp, item.leadingGcdEvent.timestamp - this.parser.pull.timestamp + TIMELINE_UPPER_MOD)}
+								/>
+							</Table.Cell>
+							<Table.Cell>
+								<Plural
+									id="core.weaving.panel-count"
+									value={item.weaves.length}
+									_1="# weave"
+									other="# weaves"
+								/>
+								&nbsp; - &nbsp;
+								{this.parser.formatDuration(item.gcdTimeDiff)}
+								&nbsp;
+								<Trans id="core.weaving.between-gcds">between GCDs</Trans>
```

Comment:
Should we split up the weave count and the time between GCDs into two separate table columns? Food for thought...

Reply:
I think I would rather remove the time after, and maybe change the Weave Info column to be actual/expected.  Even in the current accordion style, there's been a fair amount of confusion caused by the "time between GCDs" value, and I wonder if it's a net value add or not.

Reply:
@AkairyuGestalter
originally I was thinking two columns, but I didn't wholly like the look
but I am also not wholly liking the one I posted above either since it lumps things together
I'm open to suggestions if you think there would be a nicer way to present the info
this is what it looks like as two columns (below screenshot)

@AzariahAeroborn
would it look better if we were descriptive in the actions instead of just saying gcds like this?

![image](https://github.com/xivanalysis/xivanalysis/assets/89435408/fc53d044-3fe8-4bd7-9982-667bba25f979)

Reply:
I like the columns this way, but not a fan of the gcd names in the third column personally, since it ends up showing the same info (which GCDs) twice.  Maybe that's just me knowing that "big skills = gcds, little skills = ogcds", which may or may not be obvious to the target audience tho.